implementation/src/main/java/io/smallrye/config/NameIterator.java

```diff
@@ -35,20 +35,6 @@ public NameIterator(final String name, final int pos) {
         with(name, pos);
     }
 
-    private StringBuilder acquireTmpBuilder() {
```

Comment:
Where's `tmp`? It doesn't seem to be deleted from `NameIterator`.

Reply:
Correct. Deleted.